docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -554,9 +554,156 @@ Payload objects with the following scopes take the form that follows:
 
 Consult the [pages section of the Channel Webhooks Guide](/docs/integrations/webhooks/events/channels#pages).
 
+## Price Lists
+The following price lists webhook events fire in response to actions that affect a specific price list:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceList/created`  | Fires when a price list is created. | 
+| `store/priceList/updated/` | Fires when a price list is updated. | 
+| `store/priceList/activated/` | Fires when a price list is activated. | 
+| `store/priceList/deactivated/` | Fires when a price list is deactivated. | 
+| `store/priceList/deleted/` | Fires when a price list is deleted. | 
+
+Payload objects with the following scopes take the form that follows:
+
+* `store/priceList/created`
+* `store/priceList/updated/` 
+* `store/priceList/activated/`
+* `store/priceList/deactivated/`
+* `store/priceList/deleted/`
+
+```json filename="Example price list created payload object" showLineNumbers copy
+{
+  "scope": "store/priceList/created",
+  "store_id": "1001197568",
+  "data": {
+    "price_list_id": 5
+  },
+  "hash": "e0c298b8097a6a2f39d17e593a9b360f5b2fef7d",
+  "created_at": 1709327895,
+  "producer": "stores/{store_hash}"
+}
+```
+The following price lists batch webhook events fire in response to actions that affect price lists:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceLists/deleted/` | Fires when price list are deleted. | 
+
+Payload objects with the following scopes take the form that follows:
+* `store/priceLists/deleted/`
+
+```json filename="Example price lists deleted payload object" showLineNumbers copy
```

Comment:
```suggestion
```json filename="Example payload object for deleting price lists" showLineNumbers copy
```